spec/integration/data_stream/_dev/test/system/config.spec.yml

```
@@ -34,6 +34,15 @@ spec:
           enum:
             - docker
           default: docker
+        type:
+          description: >
+            "Elastic Agent image to be used for testing. Setting default will be used the
+            same Elastic Agent image as the stack."
```

Quotes not needed here.

Andd add here the description for the `complete` image too?
```suggestion
            Elastic Agent image to be used for testing. Setting `default` will be used the
            same Elastic Agent image as the stack. Setting `complete` will use the "complete"
            image, that includes all supported collectors, a web browser and the required runtime
            for synthetic testing.
```